.github/workflows/cross.yml

```diff
@@ -58,27 +58,36 @@ jobs:
       - name: Cross-compilation w/ bindgen
         run: cross test -p aws-lc-rs --release --features bindgen,unstable --target ${{ matrix.target }}
 
-  aws-lc-rs-platform-build:
+  aws-lc-rs-ios-aarch64:
     if: github.repository == 'aws/aws-lc-rs'
-    name: Cross-platform build
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        rust: [ stable ]
-        os: [ macos-13-xlarge ]
+    name: iOS aarch64 cross-platform build
+    runs-on: macos-13-xlarge
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: 'recursive'
       - uses: dtolnay/rust-toolchain@master
         id: toolchain
         with:
-          toolchain: ${{ matrix.rust }}
+          toolchain: stable
           target: aarch64-apple-ios-sim
-      - name: Set Rust toolchain override
-        run: rustup override set ${{ steps.toolchain.outputs.name }}
       - name: Install bash
         run: brew install bash
       - name: iOS Simulator Runner
         run: ./scripts/ci/ios-simulator-runner.sh
```

Comment:
NP: Might be worth renaming this file to make it clear that it only tests the iOS aarch64 simulator. Just double checking, `cargo test --features bindgen,unstable --target x86_64-apple-ios` isn't runnable on an M1 machine right?